.travis.yml

```diff
@@ -13,12 +13,12 @@ jobs:
       if: type IN (pull_request)
       install: true
       script:
-      - docker build -t microsoft/terraform-test:$TERRAFORM_VERSION . --build-arg tfver=$TERRAFORM_VERSION
+      - docker build -t microsoft/terraform-test:$TERRAFORM_VERSION . --build-arg tfver=$TERRAFORM_VERSION --build-arg gover=$GOLANG_VERSION
     - stage: push docker image
       if: (branch = master) AND (type IN (push, api, cron))
       install: true
       script:
-      - docker build -t microsoft/terraform-test:$TERRAFORM_VERSION -t microsoft/terraform-test:latest . --build-arg tfver=$TERRAFORM_VERSION
+      - docker build -t microsoft/terraform-test:$TERRAFORM_VERSION -t microsoft/terraform-test:latest . --build-arg tfver=$TERRAFORM_VERSION --build-arg gover=$GOLANG_VERSION
       - docker login -u="$DOCKER_USER" -p="$DOCKER_PASSWORD"
       - docker push microsoft/terraform-test:$TERRAFORM_VERSION
```

Comment:
Since we introduce Go SDK, we probably need to have its own version instead of aligning with terraform runtime always.

Reply:
Because we should not hardcode Go version, I have already added it to environment variables which is now 1.10.3. So what kind of changes is needed here?